src/patternfly/base/patternfly-variables.scss

```diff
@@ -17,19 +17,19 @@
   @include dark.pf-v6-tokens;
 }
 
-// :root:where(.pf-v6-theme-high-contrast) {
-//   @include highcontrast.pf-v6-tokens;
-// }
+:root:where(.pf-v6-theme-high-contrast) {
+  @include highcontrast.pf-v6-tokens;
+}
 
 @media (prefers-contrast: more) {
```

Comment:
You'll need to take out this block and the one below it that applies the CSS using `prefers-contrast` https://github.com/patternfly/patternfly/blob/f3acfe220a7b2b2756103adef041652608b324d0/src/patternfly/base/patternfly-variables.scss#L34-L38